### ISSUE TITLE
no-inconsistent-return: bad issue location

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarTS#178]
For issue location we often use first token of function declaration. It looks weird for methods with modifier keywords.
```
class A {
  public inconsistent(p: boolean) {
//^^^^^^  {{Refactor this function to use "return" consistently}}
    if (p) {
      return true;
    }
  }
}
```

**Comment by stas-vilchik** [comment]
fixed by #382

**Comment by vilchik-elena** [comment]
duplicate #374
